staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -704,7 +703,7 @@ func TestErrorFilePatternBuilder(t *testing.T) {
 }
 
 func setupKustomizeDirectory() (string, error) {
-	path, err := ioutil.TempDir("/tmp", "")
+	path, err := os.MkdirTemp("/tmp", "")
```

Comment:
I think ALL of these calls to `os.MkdirTemp` can pass "" as the first arg, no?

Reply:
yes, it can pass "" as the first arg,  and I updated it. PTAL @thockin